vsphere/datadog_checks/vsphere/api.py

```diff
@@ -452,8 +456,24 @@ def get_vsan_metrics(self, cluster_nested_elts, entity_ref_ids, id_to_tags, star
             discovered_metrics = vsan_perf_manager.QueryVsanPerf(vsan_perf_query_spec, cluster_reference)
             for entity_type in discovered_metrics:
                 for metric in entity_type.value:
+                    if metric.metricId.label in ('used', 'total'):
+                        if metric.values != 'None':
+                            relabelMap = {
+                                'used': 'size_used',
+                                'total': 'size_total',
+                            }
+                            redapl_cluster_data[relabelMap[metric.metricId.label]] = int(metric.values.split(',')[-1])
                     metric.metricId.dynamicProperty.append(
                         id_to_tags[entity_type.entityRefId.replace("'", "").split(':')[-1]]
                     )
+            cpuCount = 0
+            for host in cluster_reference.host:
+                for vm in host.vm:
+                    cpuCount += vm.summary.config.numCpu
+            # TODO: get cost multiplier since cost = cpuCount * multiplier
```

Comment:
when will we be able to do this?

Reply:
Still waiting on product input on this, so yes, this pr won't be submitted until this todo is resolved.